CHANGELOG.md

```diff
@@ -9,6 +9,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 None
 
+<a name="3.6.3"></a>
+
+## [3.6.3] - 2024-07-31
```

Comment:
I would bump this to 3.7.0 since we are adding a new feature to the SDK. Python SDK also is following this same format so we should follow this format throughout all other SDK pagination feature additions to the other languages.